### ISSUE TITLE
--version flag does not work the the 3.7.0 version of the Helm CLI

### ISSUE THREAD

**Issue by tardis4500** [chartmuseum/helm-push#111]
helm version:
version.BuildInfo{Version:"v3.7.0", GitCommit:"eeac83883cb4014fe60267ec6373570374ce770b", GitTreeState:"clean", GoVersion:"go1.16.8"}

helm plugin list push:
NAME    VERSION DESCRIPTION
push    0.9.0   Push chart package to ChartMuseum

helm push test-1.0.0.tgz cpsi --version=1.0.0-1:
Error: unknown flag: --version

Note that the option is no longer even shown in the:

Upload a chart to a registry.

If the chart has an associated provenance file,
it will also be uploaded.

Usage:
  helm push [chart] [remote] [flags]

Flags:
  -h, --help   help for push

Global Flags:
      --debug                       enable verbose output
      --kube-apiserver string       the address and the port for the Kubernetes API server
      --kube-as-group stringArray   group to impersonate for the operation, this flag can be repeated to specify multiple groups.
      --kube-as-user string         username to impersonate for the operation
      --kube-ca-file string         the certificate authority file for the Kubernetes API server connection
      --kube-context string         name of the kubeconfig context to use
      --kube-token string           bearer token used for authentication
      --kubeconfig string           path to the kubeconfig file
  -n, --namespace string            namespace scope for this request
      --registry-config string      path to the registry config file (default "/home/jgs10292/.config/helm/registry.json")
      --repository-cache string     path to the file containing cached repository indexes (default "/home/jgs10292/.cache/helm/repository")
      --repository-config string    path to the file containing repository names and URLs (default "/home/jgs10292/.config/helm/repositories.yaml")

**Comment by tardis4500** [comment]
I think the issue is that this version of helm introduces its own "push" command which hides the plugin.

**Comment by boimler** [comment]
You're right. We ' ve the same problem here. Please adapt the possibility to use this plugin besides the new helm push @ 3.7.0

Best regards.
